### DISCUSSION TITLE
Environmental

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#907] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/environmental
+ 6 new trees

**Comment by TheFastHero** [comment]
1.20.1 as well, please

**Comment by thatsacat** [comment]
1.21.1 Version of Environmental is out now add it to the tags please

**Comment by szunakabeats** [comment]
1.21.1 neoforge is the best version i think :) how is the progress ?

**Comment by szunakabeats** [comment]
hello, just wanted to check in..

**Comment by Greenblyat** [comment]
1.20.1 forge for me, hoping for it
